src/script/view_model/content/LegalHoldModalViewModel.ts

```diff
@@ -134,7 +134,7 @@ export class LegalHoldModalViewModel {
           response.client.id,
           response.last_prekey,
         );
-        fingerprint = splitFingerprint(fingerprintString);
+        fingerprint = fingerprintString;
```

Comment:
Just set fingerprint to the result in line 132.

Reply:
🤦‍♀️ of course